src/activation.jl

```diff
@@ -17,6 +17,15 @@ const sigmoid = σ
   σ(x::ForwardDiff.Dual{T,Float32}) where T = σ_stable(x)
 end
 
+"""
+   hardσ(x) = max(0, min(1.0, 0.2 * x + 0.5))
+
+Segment-wise linear approximation of sigmoid
+Note: It should not be used with Regression tasks.
```

Comment:
Can you give a reason why you've added this note?  Is there a paper that explains this?

Reply:
I have added this note because the errors will be higher in case of Regression tasks while using piecewise approximations and also it has been observed to get better results on DL Binary classification tasks.

Reply:
Another thing is that here I have implemented it the way it has been implemented in Tensorflow as the coefficient of x is 0.2. In original paper, it was taken as 0.5.(https://arxiv.org/pdf/1511.00363.pdf). Is that ok?

Reply:
It should probably be a parameter with a default value (the TF value is fine, I think).

Reply:
Ok, I will change it to a parameter

Reply:
I don't think we should include that note, at least not without a publication explaining why.  I don't think we have enough proof to put that strong of an assertion into the source code.

Reply:
Ok, I will remove that